### DISCUSSION TITLE
Can't access NAS storage only Container itself

### DISCUSSION THREAD

**Discussion by ddawson100** [jlesage/docker-qdirstat#13] [Q&A]
I've just installed this and accessing it via VNC. I can open File > Select Directory and I see two Places and Mounted Filesystems: /config (no size or FS listed) and /config 10.7 TB ext4, the latter of which must be my QNAP storage. When I select the latter, on the right side, under Path, I see bin, config, defaults, dev, etc,... If I choose the root I do get some result/analysis but it's only 184.7 MB which is about the size of the container image of 175.35 MB. I wanted some way to access/scan the QNAP storage. Am I doing something wrong?
![QDirStat](https://github.com/user-attachments/assets/afe1290f-f394-4977-bac5-540235bbf2e4)

**Comment by jlesage** [comment]
How did you created the container ? Normally, you should have setup a volume mapping for `/storage` and all the data would be found under `/storage` inside the container.

**Reply by ddawson100** [reply]
Thanks for the reply. In the QNAP you can use YAML to configure the container. I've used this:

version: '3'
services:
  qdirstat:
    container_name: QDirStat
    image: "jlesage/qdirstat"
    volumes:
    - "/Backups"
    ports:
    - "5800:5800"

For volumes I've also left it at "/" and there's no difference. It seems to be adding a directory with the name I use under volumes so in this example, it created a volume called Backups. I used this because it's the name of the directory on the QNAP that stores our backups, of course.

**Reply by jlesage** [reply]
Is this a docker compose file ? How to you create the container with this YAML ?

**Reply by ddawson100** [reply]
In the QNAP you go to Container Station > Create > Create Application. From there a Create Application box pops up and then I pasted above into it. Sorry, I'm not super familiar with Docker.

![Docker](https://github.com/user-attachments/assets/b482a709-26ed-493e-b381-c2d452fc3de0)
